packages/firefuel/lib/src/firefuel_collection.dart

```diff
@@ -78,13 +78,37 @@ abstract class FirefuelCollection<T extends Serializable>
   }
 
   @override
-  Stream<List<T>> listenWhere(List<Clause> clauses, {int? limit}) {
-    final filteredQuery = _queryFromClauses(clauses);
-    final query = limit == null ? filteredQuery : filteredQuery.limit(limit);
+  Stream<List<T>> listenOrdered(List<OrderBy> orderBy) {
+    return ref.orderBy(orderBy).snapshots().toListT();
```

Comment:
I really don't see how this is working. The `orderBy` function is looking for you to pass it a single field but you're passing it a `List<OrderBy>`. Did you add tests to make sure that it was returning results that were ordered the way you expected?

Reply:
Ahhhh yeah, this won't work.
I did write tests that passed. I wonder if I missed a test that I needed to write? 🤔